docs/learn-about-neptune/team-management.rst

```diff
@@ -1,14 +1,138 @@
+
+
 .. contents::
     :local:
     :depth: 1
     :backlinks: top
 
 .. _how-to_team-management_create-organization:
 
-Create an organization
+Organization Types
+------------------
+
+.. _core-concepts_organization-types:
+
+An organization is a way to centrally manage projects, users and subscriptions.
+
+Neptune has two organization types: individual and team.
+
+Individual
+^^^^^^^^^^
+* Each user is assigned individual organization with ``username`` as an organization name.
+* The user is the only member of this organization but may :ref:`invite collaborators <how-to_team-management_invite-to-project>` to projects.
+* The user can create an unlimited number of projects in the individual organization.
+
+Team
+^^^^
+* A team organization comes in handy when an entire team needs to be managed centrally.
+* Once :ref:`created <how-to_team-management_create-organization>`, a team organization can be managed by the :ref:`organization Admin <core-concepts_user-roles_organization-admin>`. This include users and subscription.
+* Only users who joined team organization can browse its content, subject to the assigned :ref:`role <core-concepts_user-roles>` in the organization or project.
+
+Learn more about :ref:`project types <core-concepts_project-types>` and :ref:`user roles <core-concepts_user-roles>`.
+
+=======
+
+User Roles
+----------
+.. _core-concepts_user-roles:
+
+There are roles in projects and in organizations.
+
+
+Roles in an organization
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+Organization roles apply only to team organizations.
+
+.. _core-concepts_user-roles_organization-admin:
+
+If you have :ref:`created an organization <how-to_team-management_create-organization>`,
+you can invite people to it. Two roles are available: Admin and Member.
+
+The following table summarizes the permissions available for the organization roles.
+
+.. csv-table::
+   :header: "Permission","Admin","Member"
+   :widths: 20, 20, 20
+
+    Access level in subscription management,edit,none
+    Access level in organization members,edit,none
+    Access to invoice data,edit,none
+    Access level for creating Projects,edit,none
+
+
+You can access organization settings by clicking **Settings**.
+
+.. figure:: ../_static/images/core-concepts/org-settings.png
+   :target: ../_static/images/core-concepts/org-settings.png
+   :alt: organization settings button
+
+
+.. note::
+
+    - An Admin of an organization is also the Owner of all projects, by default.
+    - Project Owners can grant project ownership to anyone in the organization.
+    - An organization must have at least one Admin, so the last remaining Admin cannot be removed from the organization.
+
+Roles in a project
+^^^^^^^^^^^^^^^^^^
+.. _core-concepts_user-roles_project-owner:
+
+The following table summarizes the permissions available for the three project roles: Owner, Contributor and Viewer.
+
+.. csv-table::
```

Comment:
This table is confusing, in my opinion.

The concept is actually quite simple:

There are 3 groups of resources in a project:
1. Its content (experiments, notebooks, wiki)
2. Project's members
3. Project's properties

Ad 1 (content):
Viewer can view anything.
Contributor can edit anything (add, run, delete, modify, etc).
Owner is the same as Contributor with regard to content.

Ad 2 (members):
Viewer can view
Contributor can view
Owner can modify (invite, remove, change roles)

Ad3 (settings):
Viewer cannot access (view or edit)
Contributor cannot access (view or edit)
Owner can modify (change visibility, name)